courses/fundamentals_of_ada/020_declarations.rst

```diff
@@ -215,10 +215,12 @@ Decimal Numeric Literals
 Based Numeric Literals
 ------------------------
 
-.. code::
+.. container:: latex_environment small
+
+ .. code::
 
    based_literal ::=
-     base # based_number [.based_number] # E [+numeral|-numeral]
+     base # base_number [.based_number] # E [+numeral|-numeral]
```

Comment:
The `based_number` is a weird way of saying "valid number for the base" but it's RM parlance, and it's also present in the decimal part (see https://www.adaic.org/resources/add_content/standards/12rm/html/RM-2-4-2.html#p2).

Maybe we should replace this whole expression by `base#numeral[.numeral]#exponent`?

Reply:
This would have the added benefit of reducing the expression size and not having to rely on the latex_environment small hack as a result

Reply:
Good idea